pephub/routers/api/v1/namespace.py

```diff
@@ -440,3 +445,15 @@ async def get_namespace_information(
     agent: PEPDatabaseAgent = Depends(get_db),
 ) -> ListOfNamespaceInfo:
     return agent.namespace.info(limit=limit)
+
+
+@namespaces.get(
+    "/stats",
+    summary="Get statistics about each namespace",
+    response_model=NamespaceStats,
+)
+async def get_namespace_information(
+    agent: PEPDatabaseAgent = Depends(get_db),
+    namespace: Optional[str] = None,
+):
+    return agent.namespace.stats(namespace=namespace)
```

Comment:
What if namespace is one that doesnt exist? Do we need to catch for `404`s?

Reply:
done